tests/checkings.test.js

```diff
@@ -0,0 +1,96 @@
+import HomePage from "../pages/HomePage";
+import SignIn from "../pages/SignIn";
+import CheckingPage from "../pages/CheckingPage"
+ 
+
+describe('Checkings', () => {
+it('Should take you to Checkings form', async ()=> {
+    await SignIn.open('bank')
+    let username = 'jsmith@demo.io'
+    let password = 'Demo123!'
+    await SignIn.logIn(username, password)
```

Comment:
Consider you'll need to login for every test in this suite, not just the first one. If not, you'll make them dependent of the order they'll be defined (which is an antipattern for tests).

Test frameworks provide a clean abstraction for dealing with this scneario (executing a piece of logic for every test on a suite) called **hooks** (in particular, the hook that fits this is the **beforeEach** hook).

Implement this login logic for this suite in a beforeEach hook

Reference: https://mochajs.org/#hooks

BTW, wdio uses lots of already existent technologies under the hood: **Mocha** is one of them, for everything having to do with test definition / execution, **Jest** for matchers/asserts, so anything you read there is valid for wdio as well.